packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -503,6 +499,186 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
             }
           });
         });
+
+        describe('NON-BASIC Tiers', () => {
+          const createAliasForNonBasicPlans = async (mode: SubscriptionTier) => {
+            const aliasAccount = await Utils.createAliasAccount(
+              mirrorNode,
+              global.accounts[0],
+              requestId,
+              initialBalance,
+            );
+
+            const hbarSpendingPlan = await hbarSpendingPlanRepository.create(mode, requestDetails, mockTTL);
+
+            await ethAddressSpendingPlanRepository.save(
+              { ethAddress: aliasAccount.address, planId: hbarSpendingPlan.id },
+              requestDetails,
+              mockTTL,
+            );
+
+            return { aliasAccount, hbarSpendingPlan };
+          };
+
+          describe('Pre-configured spending plans', async () => {
+            const expectedNonBasicPlans = {
```

Comment:
Please, instead of duplicating the content of `testSpendingPlansConfig.json` here, load the file and parse it into `expectedNonBasicPlans` dynamically. 

In this way we can only modify the JSON file to extend the tests with more plans and we don't need to modify the code of the tests.

Reply:
This is a very good point! Updated!